src/main/java/br/com/alura/comex/Menu.java

```diff
@@ -0,0 +1,39 @@
+
+package br.com.alura.comex;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import br.com.alura.comex.processadores.TipoDeProcessador;
+import br.com.alura.comex.relatorios.TipoDeRelatorio;
+
+public class Menu {
+
+	public static void exibirMenu()  throws Exception {
+		
+		Scanner sc = new Scanner(System.in);
+		
+		System.out.println(
+				"Bem-vindo ao Comex\n\nDigite a opção correspondente ao tipo de arquivo que será lido: \n1 - CSV;\n2 - JSON;\n3 - XML");
+		String opt = sc.next();
+		
+		System.out.println("Digite o nome do seu arquivo (sem extensão)");
+		
+		String nomeArquivo = sc.next();
+		
+		List<Pedido> pedidos = new ArrayList<>();
+
+		pedidos = TipoDeProcessador.getTipoDeProcessador(opt, nomeArquivo, pedidos);
```

Comment:
Não precisa passar essa lista de pedidos para o `TipoDeProcessador.getTipoDeProcessador`. Aí, é só recebê-la como retorno desse método!

```suggestion
		List<Pedido> pedidos = TipoDeProcessador.getTipoDeProcessador(opt, nomeArquivo);
```